### PR TITLE
Add PR backport script.

### PR DESCRIPTION
**Pull request by Zimmi48**
Following @maximedenes' example, here is the script that I use to backport PR.
Suggestions for improvement are welcome.

### CODE REVIEW

**Comment by gares** [comment]
My shell is bash, but others may have a different one.
The `SHELL` variable hods that piece of info. I've no clue if calling a script alters it.

Otherwise `getent passwd $USER | cut -d : -f 7-` seems to give a reasonable value

**Comment by Zimmi48** [comment]
If you mean: I should replace `#!/bin/env bash` by `#!/bin/sh` I would be fine with it but because I don't know the POSIX specification, I never know if I use bash specific features. Here the code is sufficiently simple that it is probably not the case but can you (or someone else) confirm?

**Comment by Zimmi48** [comment]
If you mean: does starting a script with `#!/bin/env bash` guarantees that the interpreter will be `bash` then the answer is yes. That is the whole point of shebangs.

File: dev/tools/backport-pr.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        done
+    fi
+    git checkout -
+
+else
+
+    echo
+    read -p "Skip directly to merging phase? [y/N] " -n 1 -r
+    echo
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
```
Comment (SkySkimmer) [inline dev/tools/backport-pr.sh]: I think `[[` and its `=~` operator are bash only.
Also where does REPLY come from?

Comment (Zimmi48) [reply dev/tools/backport-pr.sh]: I just imitated Maxine's script here. I used to do it differently. I can go back to the other method if it makes the script more generic.

**Comment by gares** [comment]
I was not talking about the shebang line, but rather the invocations of `bash` inside the loop

**Comment by maximedenes** [comment]
@gares please make explicit if you request changes or not.

**Comment by maximedenes** [comment]
Ok, @gares told me offline that he did not request changes, so merging.

**Comment by Zimmi48** [comment]
> I was not talking about the shebang line, but rather the invocations of bash inside the loop

Oh indeed, I didn't think about this.

**Pull request merged**